app/views/posts/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>This is a list of all the posts in the app.</h1>
+<h1>Posts</h1>
+
+<div id="posts">
+  <% @posts.each do |post| %>
+    <%= render post, comments: post.comments %>
+  <% end %>
+</div>
```

Comment:
- [ ] Please your user index file should match the given routes `http://127.0.0.1:3000/users/1/posts` [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)